js/app.js

```diff
@@ -0,0 +1,170 @@
+}, {
+    //Question 2
+    question: "Which beautiful country has 10 times more sheep than people?",
+    imageUrl: 'images/nz-sheep.jpg',
+    choices: ["Ireland", 'South Africa', 'New Zealand'],
+    correct: 2,
+    details: "New Zealand: Also home to the flightless and endangered kiwi bird."
+}, {
+    //Question 3
+    question: "This popular restaurant street can be found in...",
+    imageUrl: 'images/guijie.jpg',
+    choices: ["Manila, Phillipines", 'Beijing, China', 'Kuala Lumpur, Malaysia'],
+    correct: 1,
+    details: "Beijing, China: Guijie (aka Ghost Street) is a popular spot for customers and vendors. "
+}, {
+    //Question 4
+    question: "This country is famous for this kind of arena.",
+    imageUrl: 'images/Ronda.jpg',
+    choices: ["United States", 'Spain', 'Brazil'],
+    correct: 1,
+    details: "Spain: This bullfighting arena is in Ronda."
+}, {
+    //Question 5
+    question: "Where can you find this majestic scene?",
+    imageUrl: 'images/us-bison.jpg',
+    choices: ["United States", 'Poland', 'Australia'],
+    correct: 0,
+    details: "United States: The North American bison is related to, but distinct from, species of buffalo found elsewhere in the world."
+}, {
+    //Question 6
+    question: "Where is this iconic building located?",
+    imageUrl: 'images/red-square.jpg',
+    choices: ["Istanbul, Turkey", 'Berlin, Germany', 'Moscow, Russia'],
+    correct: 2,
+    details: "Moscow, Russia: This is the famous Red Square."
+}, {
+    //Question 7
+    question: "Which country offers this view?",
+    imageUrl: 'images/patagonia.jpg',
+    choices: ["Chile", 'New Zealand', 'Canada'],
+    correct: 0,
+    details: "Chile: Patagonia is an adventure lover's paradise."
+}, {
+    //Question 8
+    question: "This very old and very famous site is found in...",
+    imageUrl: 'images/angkor-wat.jpg',
+    choices: ["India", 'Cambodia', 'China'],
+    correct: 1,
+    details: "Cambodia - Angkor Wat was built in the early 12th century."
+}, {
+    //Question 9
+    question: "Bright colors and narrow alleys are common sights in this country.",
+    imageUrl: 'images/Morocco.jpg',
+    choices: ["Thailand", 'Croatia', 'Morocco'],
+    correct: 2,
+    details: "Morocco: Chefchaouen in particular is known for its blue buildings."
+}, {
+    //Question 10
+    question: "If you were mooring your sailboat in this harbor, what nation would you be in?",
+    imageUrl: 'images/victoria-harbor.jpg',
+    choices: ["England", 'Canada', 'Australia'],
+    correct: 1,
+    details: "Canada: Victoria Harbor is a popular entrance to Vancouver Island for those coming by boat or seaplane."
+}];
+//Other Variables
+var questionNum = 0;
+var numCorrect = 0;
+var questionTotal = questions.length;
+
+//Step 2 Declare Functions
+//Show questions
+//alert('I have activated the startQuiz function');
+function displayQuestions() {
+    $('#userChoices').empty();
```

Comment:
any of these selectors that are used frequently should be stored in a variable to make the code more performant